pytripgui/canvas_vc/plotter/images/patient_image_base.py

```diff
@@ -53,3 +60,28 @@ def is_present(self) -> bool:
         Returns True if image is present and False if image is absent
         """
         return self._image is not None
+
+    def calculate_extent(self, data: PlotDataBase):
+        """
+        Returns extent for passed data.
+        """
+        # get minimal X, Y and Z coordinates
+        cube: Cube = data.cube
+        min_x_mm, min_y_mm, min_z_mm = cube.indices_to_pos([0, 0, 0])
+        # get maximal X, Y and Z coordinates
+        # 'data.cube.dimz - 1' is described in https://github.com/pytrip/pytrip/issues/592
+        max_x_mm, max_y_mm, max_z_mm = cube.indices_to_pos([cube.dimx, cube.dimy, cube.dimz - 1])
+        plane = data.projection_selector.plane
+        # depending on plane, return proper list those above
+        # extent = [horizontal_min, horizontal_max, vertical_min, vertical_max]
+        # "Transversal" (xy)
+        if plane == "Transversal":
+            return [min_x_mm, max_x_mm, min_y_mm, max_y_mm]

```

Comment:
```suggestion
            return (min_x_mm, max_x_mm, min_y_mm, max_y_mm)

```

I'd reccomend immutable tuples as return type